package.json

```diff
@@ -55,7 +58,8 @@
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
     "react-ga": "^2.5.6",
-    "react-styleguidist": "^8.0.6",
+    "react-styleguidist": "^9.0.0",
+    "react-test-renderer": "^16.12.0",
```

Comment:
We need to merge `master` into this branch. Looks like this dependency version is wrong. `react`, `react-dom`, and `react-test-renderer` need to match on their versions. I think this is fixed in `master` and I found I didn't need to mess with other dependencies. I think the only thing that is missing from `master` is `@testing-library/react-hooks`, so I am not sure if that messes with these other testing libraries.

Reply:
It is messing with it, `renderer` was too far behind to work with `hooks` included. I believe I've fixed the deps and lockfile after quite a bit of faffing about. 😅

Reply:
Looks good. I thought `react` and any `react-` libraries were supposed to have their versions the exact same. But I couldn't find much to back that up. I just have a recollection of React devs encouraging or fixing bugs by doing that. I guess we should just let dependency validation do its own magic.